pkg/broker/broker.go

```diff
@@ -470,6 +470,14 @@ func (a AnsibleBroker) Catalog() (*CatalogResponse, error) {
 		if err != nil {
 			log.Errorf("not adding spec %v to list of services due to error transforming to service - %v", spec.FQName, err)
 		} else {
+			// Bug 1539542 - in order for async bind to work,
```

Comment:
I don't think that we need this in an else block. Just a style thing, surprised if this passes lint.

Reply:
it was already there so I didn't want to muck with the existing code. I'll fix it though.